helpers/labels/service_public.json

```diff
@@ -31,5 +31,6 @@
     "CGSS DE LA GUYANE": "315190769",
     "CGSS DE LA REUNION": "314635483",
     "CGSS DE MAYOTTE": "479281719",
-    "CENTRE DE GESTION PAM": "314635483"
+    "CENTRE DE GESTION PAM": "314635483",
+    "BPI France": "320252489"
```

Comment:
What do you think about adding a README.md in this folder to explain the source of USRSSAF rows and the transformation you applied? So it is clear for anyone where the data comes from?